test/e2e/docs/debugging.md

```diff
@@ -61,11 +63,50 @@ eg.
 DEBUG=mocha:* node_modules/.bin/mocha specs/wp-manage-domains-spec.js
 ```
 
-## Jest and Playwright
+## Playwright/Jest
 
-Our Playwright E2E tests run on Jest. Fortunately, Jest has fanstastic documentation on setting up debuggers, which can be found here: [https://jestjs.io/docs/troubleshooting](https://jestjs.io/docs/troubleshooting).
+### Debug instance
 
-A couple notes:
+While developing tests and/or debugging flakey e2e tests, it is often helpful to have a browser window open with Playwright hooked in.
 
-- If using VSCode, setting up the debugger using the attaching method is often easier (as opposed to directly launching Jest in the `launch.json`). The attach configuration gives you more control over which script you want to run.
-- The E2E tests use the Jest binary that is installed at the root level of the repo. Put all together, if you were currently in the `e2e` directory, the command to run a single spec would look like `node --inspect-brk ../../node_modules/.bin/jest --runInBand specs/specs-playwright/wp-auth__canary-spec.ts`
+1. set Jest timeout to a long value:
+
+```typescript
+// in-code
+jest.setTimeout(1000000000);
+
+// in default.json
+"jestTimeoutMS": 1600000000,
+```
+
+2. launch Playwright with timeout disabled and verbose logs:
+
+```bash
+export PWDEBUG=1 DEBUG=pw:api yarn jest <spec>
```

Comment:
I'd recommend passing `--runInBand` as well. Also, timeout can be changed from the CLI, using `--testTimeout=<number>`

Reply:
Incorporated suggestion in 6b898b66843c11a5c2422dd01733eb88dbe1ace9.